index.js

```diff
@@ -0,0 +1,133 @@
+/*
+        @@@@@@@            @@@@@@@         @@
+      @@       @@        @@       @@      @@@
+    @@   @@@ @@  @@    @@   @@@ @@  @@   @@@@@@ @@   @@@  @@ @@@      @@@@
+   @@  @@   @@@   @@  @@  @@   @@@   @@   @@@   @@   @@@  @@@   @@  @@@   @@
+   @@  @@   @@@   @@  @@  @@   @@@   @@   @@@   @@   @@@  @@@   @@  @@@@@@@@
+   @@  @@   @@@  @@   @@  @@   @@@  @@    @@@   @@   @@@  @@@   @@  @@@
+    @@   @@@ @@@@@     @@   @@@ @@@@@      @@@    @@@ @@  @@@@@@      @@@@@
+      @@                 @@                           @@  @@
+        @@@@@@@            @@@@@@@               @@@@@    @@
+                                                          */
+//. # sanctuary-type-identifiers
+//.
+//. A type is a set of values. The Boolean type, for example, is the type
+//. comprising `true` and `false`. A value may be a member of several types.
+//. `42`, for example, is a member of Number, PositiveNumber, Integer,
+//. PositiveInteger, and many other types.
+//.
+//. In certain situations it is useful to divide JavaScript values into
+//. non-overlapping types. The language provides two constructs to support
+//. this: the [`typeof`][1] operator, and [`Object.prototype.toString`][2].
+//. Each has pros and cons, but neither supports user-defined types.
+//.
+//. This package specifies an [algorithm][3] for deriving a type identifier
+//. from any JavaScript value, and exports an implementation of this algorithm.
```

Comment:
> exports an implementation of this algorithm

Nowhere in the readme does it show how to use this exported algorithm, only how to comply with it. I think it'd be helpful to have an example of its usage somewhere.